app/src/main/res/layout/activity_location.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LocationActivity">
+
+    <Button
+        android:id="@+id/getlocation_location_button"
```

Comment:
I know it is just a minor detail, but we went through all the ids to match the naming convention, it would be a shame to have new layouts not meet the naming conventions.
For all the ids in this layout several points are to fix:

- `compositeName` instead of `composite_name`. ex: `currentLocation` instead of `current_location`
- the activity name is capital. ex: `Location`, not `location`
- `viewType` instead of `viewtype`. ex: `textView`, not `textview`

Reply:
Oh ok. But I don't think this match the conventions in the README file.